creator-node/compose/docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
       - delegateOwnerWallet=${delegateOwnerWallet}
       - delegatePrivateKey=${delegatePrivateKey}
       - creatorNodeEndpoint=${creatorNodeEndpoint}
+      - triggerSyncOnWrite=${triggerSyncOnWrite}
```

Comment:
where is this set to false for local testing? maybe i missed

Reply:
is true by default if not set, set manually in host environment prior to startup